.eslintrc

```diff
@@ -151,7 +151,7 @@ rules:
   no-implicit-coercion: [1]
   no-implicit-globals: [0]
   no-implied-eval: [2]
-  no-inline-comments: [0]
+  no-inline-comments: [2]
```

Comment:
Redoing the changes vs hard-to-resolve and easy-to-mess conflicts? I think it's the former.